<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mgrsDialog</class>
 <widget class="QDialog" name="mgrsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MGRS Settings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>MGRS precision</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="mgrsPrecisionSpinBox">
       <property name="maximum">
        <number>5</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>Coordinate prefix</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="prefixLineEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="suffixLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Coordinate suffix</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>GZD line color</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QgsColorButton" name="lineColorButton">
       <property name="color">
        <color>
         <red>1</red>
         <green>1</green>
         <blue>1</blue>
        </color>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>GZD font color</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QgsColorButton" name="fontColorButton">
       <property name="color">
        <color>
         <red>1</red>
         <green>1</green>
         <blue>1</blue>
        </color>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="addSpacesCheckBox">
     <property name="text">
      <string>Add spaces to MGRS coordinates</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="markerCheckBox">
     <property name="text">
      <string>Show a marker on QGIS map at the captured location</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="persistentMarkerCheckBox">
     <property name="text">
      <string>Use persistent zoom to marker</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsColorButton</class>
   <extends>QToolButton</extends>
   <header>qgscolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>mgrsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>mgrsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
